examples/src/fibonacci/fib2/prover.rs

```diff
@@ -84,7 +84,7 @@ where
     fn new_evaluator<'a, E: FieldElement<BaseField = Self::BaseField>>(
         &self,
         air: &'a Self::Air,
-        aux_rand_elements: AuxTraceRandElements<E>,
+        aux_rand_elements: Option<AuxRandElementsProver<Self, E>>,
```

Comment:
nit: could this be: `aux_rand_elements: Option<()>` since we set them to `()` in `Air`?

Reply:
Fixed this one and all other relevant ones